data/assets/examples/rotation/timelinerotation/no-interpolate.asset

```diff
@@ -0,0 +1,49 @@
+-- No Interpolation
+-- This asset creates a SceneGraphNode that only displays coordinate axes. The rotation of
+-- coordinate axes are determined by a timeline of individual rotations. These rotations
+-- are keyframes that are used to change between different orientations. This example
+-- transitions between three rotations. As the interpolation between entries is disabled,
```

Comment:
This sentence is a bit unclear to me: "As the interpolation between entries is disabled,
-- the orientation will change abruptly at the provided times."

Could we clarify somehow?